exercises/two-fer/introduction.md

```diff
@@ -0,0 +1,8 @@
+# Introduction
+
+In some English accents, when you say "two for" quickly, it sounds like "two fer".
+Two-for-one is a way of saying that if you buy one, you also get one for free.
+So the phrase "two-fer" often implies a two-for-one offer.
+
+Imagine a bakery that has a Valentine's Day offer where you can buy two cookies for the price of one ("two-fer one!").
```

Comment:
```suggestion
Imagine a bakery that has a special opening day offer where you can buy two cookies for the price of one ("two-fer one!").
```

Reply:
I am probably being overly picky here, but Valentine's Day isn't universally celebrated.
Other options might be birthday or holiday - or even a day of the week.

Reply:
I like "special holiday offer"

Reply:
Yepp, I don't feel strongly. How about "holiday offer"?